tests/metrics/test_bleurt20.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -eu #o pipefail
+pairs=(en-de en-ru)
+
+# Skip if no CPU found
+if [ ! $MRT_MARIAN_USE_CPU ]; then
+    exit 100
+fi
+
+readonly metrics=$MRT_MODELS/metrics
+readonly max_segs=16
+CPU_ARGS="--cpu-threads 8 -w 8000"
+#GPU_ARGS="--devices 0"  # one gpu
+MARIAN_ARGS="$CPU_ARGS --width 4 --like bleurt --mini-batch 4 --maxi-batch 256 --max-length 512 --max-length-crop true --average skip"
+
+
+for pair in ${pairs[@]}; do
```

Comment:
I would keep one language pair per test because it simplifies code and make it simpler to debug from the generated `.log` files, which just contain `bash -x`.